gensrc/thrift/Planner.thrift

```diff
@@ -65,6 +65,13 @@ struct TPlanFragment {
   // sink) in a single instance of this fragment. This is used for an optimization in
   // InitialReservation. Measured in bytes. required in V1
   8: optional i64 initial_reservation_total_claims
+  20: optional list<TGlobalDict> global_dicts
+}
+
+struct TGlobalDict {
+    1: optional i32 columnId
+    2: optional list<string> strings
```

Comment:
binary is better?

Reply:
Why binary is better.

Reply:
String's content is character, binary's is byte.
What we needs here is binary. For example, we can use binary to store int64.

Reply:
We use `HashMap<String, Integer>` in FE and BE, use string, we handle it easily in FE and BE both. if use binary, the will have Little-Endian and Big-Endian issue. The map is small, so I think there is no performance issue.

Reply:
It's not about performance. Rather, it is about expressiveness.Our dictionary encoding can be used not just for strings, but for other types. For example, in the future we will support dictionary encoding of int64_t.
And with regard to the big endian or little endian, it is not a question of whether to use string or binary.